.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
nit: thank the user (eg "Thank you for taking the time to file a bug report.") and also maybe ask to look at existing open issues

Reply:
Yeah good point, added this in.